lib/verbalize/action.rb

```diff
@@ -17,8 +17,19 @@ def self.included(target)
 
     module ClassMethods
       def input(*required_keywords, optional: [])
+        # treat `input` as an accessor if nothing is passed in.
+        return @inputs || [] if required_keywords.empty? && optional.empty?
```

Comment:
I would prefer to have the input introspection be only defined as **.inputs**, and not mix together defining inputs and listing inputs.